test/monitor.js

```diff
@@ -293,6 +302,8 @@ describe('Monitor', () => {
                         expect(monitor._server.listeners('internalError')).to.have.length(0);
                         expect(monitor._server.listeners('tail')).to.have.length(0);
                         expect(monitor._server.listeners('stop')).to.have.length(0);
+                        expect(Wreck.listeners('request')).to.have.length(0);
+                        expect(Wreck.listeners('response')).to.have.length(0);
```

Comment:
Since I added the beforeEach() hook that removes all listeners from Wreck before each test, I can now test that monitor.stop() removes the listeners from Wreck.